rust/blockstore/src/types/flusher.rs

```diff
@@ -31,4 +31,11 @@ impl BlockfileFlusher {
             BlockfileFlusher::ArrowBlockfileFlusher(flusher) => flusher.id(),
         }
     }
+
+    pub fn total_keys(&self) -> Option<u64> {
```

Comment:
nit: can this be pub(crate) instead of pub?

Reply:
I don't believe so, it's used in `rust/worker/src/segment/record_segment.rs`